### PR TITLE
Add devcontainer file

### PR DESCRIPTION
**Pull request by eddumelendez**

### CODE REVIEW

File: .devcontainer/devcontainer.json [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+	"features": {
+        "ghcr.io/devcontainers/features/git:1": {},
+        "ghcr.io/devcontainers/features/github-cli:1": {},
+        "ghcr.io/meaningful-ooo/devcontainer-features/homebrew:2": {},
+        "ghcr.io/devcontainers/features/java:1": {
+            "version": "none",
+            "installMaven": "false",
+            "installGradle": "false"
+        },
+        "ghcr.io/devcontainers/features/docker-from-docker:1": {},
```
Comment (HofmeisterAn) [inline .devcontainer/devcontainer.json:45]:
```suggestion
        "ghcr.io/devcontainers/features/docker-from-docker:2": {},
```

**Review by HofmeisterAn — APPROVED** [review]

File: .devcontainer/devcontainer.json (hunk 1 above)
Comment (eddumelendez) [reply .devcontainer/devcontainer.json]: AFAIK, there is no version 2. I just noticed it has been move to [docker-outside-of-docker](https://github.com/devcontainers/features/blob/main/src/docker-outside-of-docker/README.md)

Comment (HofmeisterAn) [reply .devcontainer/devcontainer.json]: There is: https://github.com/devcontainers/features/blob/f7314671e1654cd47263a09b134ddd0286f9eccc/src/docker-in-docker/devcontainer-feature.json#L3.

Comment (HofmeisterAn) [reply .devcontainer/devcontainer.json]: Sorry, in the first commit I mixed it up with `docker-in-docker:2`.

**Pull request merged**